DigitalOffice/Tests/CompanyServiceTests/Repositories/CompanyRepositoryTests.cs

```diff
@@ -41,6 +57,17 @@ public void CleanDb()
             }
         }
 
+        #region GetCompaniesList
+        [Test]
+        public void ShouldRightGetCompaniesList()
+        {
+            var actualCompaniesList = repository.GetCompaniesList();
+
+            var expectedCompaniesList = dbContext.Companies.ToList();
+            SerializerAssert.AreEqual(expectedCompaniesList, actualCompaniesList);
```

Comment:
Verify state of db. Something like that should work: `Assert.That(dbContext.Companies, Is.EquivalentTo(new List<DbCompany>{dbCompanyInDb})`

Reply:
Fixed?